src/Compilers/CSharp/Portable/BoundTree/Constructors.cs

```diff
@@ -523,10 +523,15 @@ public static BoundBlock SynthesizedNoLocals(SyntaxNode syntax, params BoundStat
         }
     }
 
-    internal partial class BoundDefaultOperator
+    internal partial class BoundDefaultExpression
     {
-        public BoundDefaultOperator(SyntaxNode syntax, TypeSymbol type)
-            : this(syntax, type.GetDefaultValue(), type)
+        public BoundDefaultExpression(SyntaxNode syntax, TypeSymbol type, bool hasErrors = false)
+            : this(syntax, type.GetDefaultValue(), type, hasErrors)
+        {
+        }
+
+        public BoundDefaultExpression(SyntaxNode syntax)
+          : this(syntax, constantValueOpt: null, type: null, hasErrors: false)
```

Comment:
No, this bound node can represent both `default` (a literal) and `default(T)` (not a literal).